pkg/minikube/perf/logs.go

```diff
@@ -34,6 +35,11 @@ func timeCommandLogs(cmd *exec.Cmd) (*result, error) {
 	// matches each log with the amount of time spent on that log
 	r := newResult()
 
+	output = strings.ToLower(output)
```

Comment:
We have already set `--output=json` in `start.go`, so we can remove all of this logic. Instead, we now want to parse the JSON output.

In `logs.go`, we need to make sure we are parsing the JSON output correctly. The only thing that needs to happen in this file is to this `log` variable:

https://github.com/kubernetes/minikube/blob/55f3a77d6f85b79ced3a5fdca99314a46d165d74/pkg/minikube/perf/logs.go#L54
 

and parse out the `Name` of the step, since `log` will be in JSON format.